cpp/include/cudf/detail/copy_if_else.cuh

```diff
@@ -116,23 +116,28 @@ __launch_bounds__(block_size) __global__
 
 }  // anonymous namespace
 
-template <typename Element, typename FilterFn, typename LeftIter, typename RightIter>
-std::unique_ptr<column> copy_if_else(data_type type,
-                                     bool nullable,
+template <typename FilterFn, typename LeftIter, typename RightIter>
+std::unique_ptr<column> copy_if_else(bool nullable,
```

Comment:
What does this parameter do? This function needs docs (I know you didn't write it, but lets leave it better than you found it).